source/server-selection/server-selection.rst

```diff
@@ -962,7 +974,10 @@ The single server is always suitable for write operations if it is available.
 Topology type: LoadBalanced
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-See the `Load Balancer Specification <../load-balancers/load-balancers.rst#server-selection>`__ for details.
+During command construction, the LoadBalancer server MUST be treated like a
+mongos and drivers MUST add a $readPreference field to the command when
```

Comment:
Is the bit about treating the load balanced server like a mongos required? Or can we just say the "drivers MUST add..." bit?